portal/server/nunjucks-configuration/filter-localiseTimestamp.ts

```diff
@@ -0,0 +1,20 @@
+import { formatInTimeZone } from 'date-fns-tz';
+
+export const filterLocaliseTimestamp = (
+  utcTimestamp: number | string,
+  format: string,
+  targetTimezone: string,
+) => {
+  if (!utcTimestamp) {
+    return '';
+  }
+  const timestamp = parseInt(String(utcTimestamp), 10);
```

Comment:
If this is getting converted to `String` should we limit `utcTimestamp` type to be `String` data type only?